Renaming test directory and fix install prestashop test - Smoke tests campaign (#15020)
PrestaShop/PrestaShop: tests/puppeteer/pages/CONVENTIONS.md

\*action\*()

> For example, the method allowing a user to login in the BO should be named: `login()`. Since it's belonging to the `login` page, it's obvious that it targets the form in the login page. However, at the test level, we don't need to know what's happening exactly in the page to allow us to `login`. All the low-level logic is handled by the page object. 

Reviewer: ```suggestion
For example, the method allowing a user to login in the BO should be named: `login()`. Since it's belonging to the `Login` page, it's obvious that it targets the form in the login page. However, at the test level, we don't need to know what's happening exactly in the page to allow us to `login`. All the low-level logic is handled by the page object. 
```

Author: @SimonGrn What's your suggestion in this file?

> \*office\*\_\*pagename\*

Reviewer: ```suggestion
\*Pagename\*
```
Following PSR-1 standards.

### Coding conventions

> Variables representing pages should always be named the same as the page they're referencing, but in **all caps**. For example, your test script might look like this:

Reviewer: ```suggestion
Variables representing pages should always be named the same as the page they're referencing, but in `camelCase`*. For example, your test script might look like this:
```

```javascript

> require('BO/login.js');

Reviewer: ```suggestion
const LoginPage = require('BO/login.js');
```
PSR-1 standards for class names

> const LOGIN = new login();

Reviewer: ```suggestion
const login = new LoginPage();
```

...

test('should login in the BO', () => {

>   LOGIN.setCredentials(EMAIL, PASSWD);

Reviewer: ```suggestion
  login.setCredentials(EMAIL, PASSWD);
```

>   LOGIN.login();

Reviewer: ```suggestion
  login.login();
```
